nerfstudio/process_data/process_data_utils.py

```diff
@@ -462,7 +462,7 @@ def downscale_images(
                 nn_flag = "" if not nearest_neighbor else ":flags=neighbor"
                 ffmpeg_cmd = [
                     f'ffmpeg -y -noautorotate -i "{image_dir / filename}" ',
-                    f"-q:v 2 -vf scale=iw/{downscale_factor}:ih/{downscale_factor}{nn_flag} ",
+                    f"-q:v 2 -vf scale=round(iw/{downscale_factor}):round(ih/{downscale_factor}){nn_flag} ",
```

Comment:
What's the reason to have `round` here? I believe the right approach is to ensure aspect ratio of the scaling, by having `setsar=1` append to the last. 

```
ffmpeg -y -noautorotate -i "{image_dir / filename}"  \
 -q:v 2 -vf scale=iw/{downscale_factor}:ih/{downscale_factor}{nn_flag},setsar=1
```

Reply:
Reverted change. The `cameras.py` rescaling now accepts the height and width, so no need for this entire hassle.